src/plots/cartesian/graph_interact.js

```diff
@@ -1370,16 +1371,40 @@ function dragBox(gd, plotinfo, x, y, w, h, ns, ew) {
 
     var dragOptions = {
         element: dragger,
+        gd: gd,
+        plotinfo: plotinfo,
+        xaxes: xa,
+        yaxes: ya,
+        doubleclick: doubleClick,
         prepFn: function(e, startX, startY) {
-            if(ns+ew==='nsew' && ((fullLayout.dragmode==='zoom') ?
-                !e.shiftKey : e.shiftKey)) {
+            fx.unhover(gd); // we want a clear plot for dragging
```

Comment:
@etpinard this is useful for zoom and pan too. I'm not sure what it will do with events though.